resmo/README.md

```diff
@@ -0,0 +1,60 @@
+## Overview
+[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
```

Comment:
As I understand it, this integration is involved with monitoring and security -- could you clarify how it helps with provisioning and maintaining servers and cloud infrastructure?

Reply:
Hi @buraizu,

We defined it at the "Monitor them in one place, run security and compliance audits, and get alerted on violations."  section. If you have any suggestions I would gladly add them.

Reply:
Hi @buraizu,

I appreciate your help, changes are merged now.